grpc/src/test/java/com/linecorp/armeria/client/grpc/GrpcClientTest.java

```diff
@@ -1499,8 +1498,6 @@ void unimplementedService() throws Exception {
             if (rpcReq != null) {
                 assertThat(rpcReq.params()).containsExactly(Empty.getDefaultInstance());
             }
-            assertThat(headers.get(GrpcHeaderNames.GRPC_STATUS)).isEqualTo(
```

Comment:
UnimplementedService is not added to the server so the `GRPC_STATUS` is not set.